tests/unit/lms/services/group_info_test.py

```diff
@@ -83,7 +83,9 @@ def group_info_svc(self, pyramid_request):
     @pytest.fixture
     def params(self):
         return {
-            field.key: f"TEST_{field.key.upper()}" for field in GroupInfo.iter_columns()
+            field.key: f"TEST_{field.key.upper()}"
+            for field in GroupInfo.iter_columns()
+            if field.key not in ["info"]
```

Comment:
Fun facts, last time I checked, using a list is more efficient than a set when the number of items is very low, as the hashing costs exceed the benefits when searching.

This would probably be fine as a "!=" though, which would be faster than both.